addons/point_of_sale/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -405,9 +405,25 @@ export class ProductScreen extends Component {
             ? this.getProductsByCategory(this.pos.selectedCategory)
             : this.products;
 
-        return products.filter((p) =>
-            unaccent(p.searchString, false).toLowerCase().includes(words)
+        const filteredProducts = products.filter((p) =>
+            unaccent(p.searchString, false)
+                .toLowerCase()
+                .includes(unaccent(words, false).toLowerCase())
```

Comment:
```suggestion
        const filteredProducts = products.filter((p) => unacccent(p.searchString).includes(words)
```
I see that unaccent already do `toLowerCase` and words is already unaccented and put to lower case so no need to do it again